### PR TITLE
Symphony producers

### PR DESCRIPTION
**Pull request by jgreben**

### CODE REVIEW

File: ReactiveProducers/ReactiveKafkaDumpProducer/src/main/resources/application.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+
+dataDir=${HOME}/Dev/data/ld4pData
+dataDir=${?ld4pData}
```
Comment (atz) [inline ReactiveProducers/ReactiveKafkaDumpProducer/src/main/resources/application.conf]: The convention for this ENV variable is now `LD4P_DATA`.  Please update to match.

File: ReactiveProducers/ReactiveKafkaUpdateProducer/src/main/resources/application.conf [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
+
+dataDir=${HOME}/Dev/data/ld4pData
+dataDir=${?ld4pData}
```
Comment (atz) [inline ReactiveProducers/ReactiveKafkaUpdateProducer/src/main/resources/application.conf]: Same here.

**Review by atz — CHANGES_REQUESTED** [review]

File: ReactiveProducers/ReactiveKafkaDumpProducer/src/main/resources/application.conf (hunk 1 above)
Comment (atz) [reply ReactiveProducers/ReactiveKafkaDumpProducer/src/main/resources/application.conf]: @jgreben Looks like that might be fixed if you just rebase.

**Comment by Maatary** [comment]
Merged the convention so all good

**Comment by Maatary** [comment]
Fixed the Convention problem

**Pull request merged**